### ISSUE TITLE
Notes from Labs testing 10th Sept

### ISSUE THREAD

**Issue by chris-nott** [odpi/egeria#1529]
Testing of the five Lab Notebooks on 10th September at egeria/open-metadata-resources/open-metadata-labs/administration/

I ran on a single (free) Kubernetes node on the IBM Cloud in London.  Numbers refer to sequence number of the preceding executable code fragment in the notebook.

metadata-server-config:
2: Open Metadata and Governance (OMAG) - ie expand the acronym
2: 'onto' is two words (ie 'on to')
12: typo 'connecto'
22: error 400 for the command: 'stewardship-action'
...and similarly later

managing-servers:
0: check 'catalogues' is American
0: expand OMAG

open-discovery-config:
title: Configurating should be Configuring!
3: 'begine'
4: need to start 'cocoMDS1' if it's not running
Set adminPlatformURL = devPlatformURL (not hard code to localhost
Ex3 fails on first post
Needs more explanation of what is happening

understanding-cohorts:
Check 'catalogues' and expand OMAG again
'a explaination' should be 'an explanation'
The last two commands needs some description, what and why you would use.

understanding-platform-services:
Check 'catalogues' and expand OMAG again

I do think you should include a Kubernetes cluster set up notebook, perhaps both IBM Cloud and OpenShift.

The README could be a fleshed out.

Two further suggestions:
1. show a view of corporate et al applications on top of the servers.  Could be a separate blog post.
2. capture architecture/design factors and guidance for consideration (eg how to lay out the servers).

**Comment by planetf1** [comment]
* Will address kubernetes on ibmcloud (as an example) setup blog -> https://github.com/odpi/egeria/issues/1203
* And for openshift - https://github.com/odpi/egeria/issues/1213

Though I may do some refactoring of these across k8s locally/in cloud (I find k8s awkward locally due to resource requirements, but it is the *easiest* to get going)

Creating a notebook for creation of a cluster in one/either of the above is something I'd hoped to do, but has a few complications. However it's interesting you felt it would be useful, it's not entirely trivial, so will be useful. I raised https://github.com/odpi/egeria/issues/1564 to track this

**Comment by mandy-chessell** [comment]
I have done a bit of investigation and I am going to use "catalog", "catalogs", "cataloged", "cataloging" as these are all valid, simplified forms favoured in modern US English.  

The spelling seems to have jumped around on both sides of the Atlantic and so I am going for the simplest spelling.

**Comment by RyanHolstien** [comment]
On the open-discovery-config Ex3 failure, this appears to be due to a typo in the URL.

 When looking at the building-a-data-catalog lab I found that this URL worked:
`http://datalake:8080/servers/cocoMDS1/open-metadata/access-services/asset-owner/users/peterprofile/assets/data-files/csv`

but the one in the open-discovery-config lab is:
`http://datalake:8080/servers/cocoMDS1/open-metadata/access-services/asset-owner/users/peterprofile/assets/files/csv`

Correcting the "files" to "data-files" should fix that.

**Comment by planetf1** [comment]
Hi @RyanHolstien I believe that notebook still needs more work - but spot on in terms of that particular error. We always welcome contributors so if you'd like to make a PR to submit that would be great. If not I can make the change.

**Comment by RyanHolstien** [comment]
Hi @planetf1 sorry for the late response, didn't get a chance to put it up until now. I linked it to this issue, hope that was okay. Please let me know if you need me to make any changes to the PR if it's not up to standards! I took a quick look through the contributor's guide and tried to follow it.

**Comment by planetf1** [comment]
@RyanHolstien no worries. I added some review comments to the PR. The line you proposed changing was spot on, and the process right too :-)

Unfortunately the same cell had another issue to correct, and when I looked at the notebook (and sorry not to mention this earlier), can see it has clearly not yet been completed - it will likely be focussed on in a few months. As such I suggested a minor cleanup just to make it clear to anyone else what the state may be?

Thanks for the contribution though !
